lang/test/org/partiql/lang/eval/visitors/SubstitutionVisitorTransformTest.kt

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright 2019 Amazon.com, Inc. or its affiliates.  All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ *  You may not use this file except in compliance with the License.
+ * A copy of the License is located at:
+ *
+ *      http://aws.amazon.com/apache2.0/
+ *
+ *  or in the "license" file accompanying this file. This file is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific
+ *  language governing permissions and limitations under the License.
+ */
+
+package org.partiql.lang.eval.visitors
+
+import com.amazon.ionelement.api.emptyMetaContainer
+import com.amazon.ionelement.api.ionInt
+import org.junit.Test
+import org.partiql.lang.ast.SourceLocationMeta
+import org.partiql.lang.ast.metaContainerOf
+import org.partiql.lang.ast.toIonElementMetaContainer
+import org.partiql.lang.domains.PartiqlAst
+import kotlin.test.assertEquals
+import kotlin.test.assertNull
+
+class SubstitutionVisitorTransformTest {
```

Comment:
These tests were originally from SubstitutionRewriterTest.kt with modifications to use PIG and the visitor transforms.